service/grails-app/views/licenseCore/_licenseCore.gson

```diff
@@ -6,7 +6,7 @@ import groovy.transform.Field
 @Field LicenseCore licenseCore
 
 // The new date formats seem to be causing issues when included here. So Exclude them from g.render and explicitly include them below.
-json g.render(licenseCore, [excludes: ['startDate','endDate'], expand: ['tags', 'customProperties', 'status', 'contacts', 'docs', 'supplementaryDocs' ]]) {
+json g.render(licenseCore, [excludes: ['startDate','endDate', 'type', 'orgs', 'amendments', 'alternateNames'], expand: ['tags', 'customProperties', 'status', 'contacts', 'docs', 'supplementaryDocs' ]]) {
```

Comment:
So. Given that this template will need updating every time we change license or amendments I think we should rejig this. Rather than 'excludes' we should swap to using 'includes' and only include anything that exists against the core template. i.e. json g.render(licenseCore, [excludes: ['startDate','endDate'], includes:['id', 'tags', 'customProperties', ...